docs/src/examples/components/Popup/Types/PopupNoFocusTrapExample.shorthand.tsx

```diff
@@ -1,36 +1,20 @@
 import React from 'react'
-import { Button, Input, Popup } from '@stardust-ui/react'
+import { Button, Input, Popup, popupBehavior } from '@stardust-ui/react'
 
 const PopupNoFocusTrapExample = () => (
-  <>
-    <Popup
-      focusTrap={false}
-      content={{
-        content: (
-          <>
-            <p>This popup DOESN'T trap focus.</p>
-            <Input icon="search" placeholder="Search..." />
-          </>
-        ),
-      }}
-      trigger={<Button icon="expand" content="Popup with no focus trap" />}
-    />
-
-    <Popup
-      content={{
-        content: (
-          <>
-            <p>
-              Focus is trapped.<br />
-              Proceed with typing in the input field:
-            </p>
-            <Input icon="search" placeholder="Search..." />
-          </>
-        ),
-      }}
-      trigger={<Button icon="expand" content="Popup that will trap focus" />}
-    />
-  </>
+  <Popup
+    /** 'popupBehavior' does not use focus trap */
+    accessibility={popupBehavior}
```

Comment:
let's maybe call it `popupNoFocusTrapBehavior` to make it more clear

Reply:
I am afraid it can be more confusing having 2 behaviors:
```popupNoFocusTrapBehavior``` and ```popupFocusTrapBehavior```, as the names could seem the same.
I think better to agree with one of these 2 options:

1. ```popupFocusTrapBehavior``` - default, with focus trap. ```popupBehavior``` - optional, without focus trap

2. ```popupBehavior``` - default, with focus trap. ```popupNoFocusTrapBehavior``` - optional, without focus trap

Currently the 1st option is implemented in the PR, but we can change it

Reply:
what about following 2 names:
- popupBehavior (without focus trap, default)
- dialogBehavior (with focus trap)

Reply:
From my own experience, I could never differentiate popup and modal :) So I would stick to more semantically descriptive name
Let me try once again:
- popupBehavior (without focus trap, default)
- popupFocusTrapBehavior (with focus trap)

Reply:
agree